Plots/Masking/mask_1.py

```diff
@@ -0,0 +1,187 @@
+"""
```

Comment:
The plots have a gap along 0 degrees longitude (wraparound). Can this be fixed? I think @jukent had a solution.

Reply:
This is fixed